### PR TITLE
Update vietnamese localization

### PR DESCRIPTION
**Pull request by Endy3032**
made the text feels more natural
some notes:
- on line 81 putting the typo on "đo (đó)" makes it a bit confusing the first time reading it, i shifted the typo to the word cái → cai
- on line 130, the original english localisation used a shorthand for "no problem" so i thought i'd use a shorthand for no in vietnamese too (không → ko)
- on line 140, most people understand the word status here and "trạng thái" sounds a bit weird even though its the correct translation

### CODE REVIEW

File: locales/vi/messages.po [hunk 1]
```diff
@@ -45,20 +46,20 @@ msgstr "❌ Đừng làm điều này"
...
+"Tương tự như:\n"
 "\n"
-"- \"Xin chào, bạn có ở đây không?\"\n"
-"- \"chào sophie - câu hỏi nhanh.\"\n"
-"- \"Bạn có một giây?\"\n"
+"- \"Xin chào, bạn có đó không?\"\n"
+"- \"chào sophie - cho hỏi nhanh nhé.\"\n"
+"- \"Bạn có rảnh chút không?\"\n"
 "- \"yt?\"\n"
 "- \"ping\"\n"
```
Comment (lts20050703) [inline locales/vi/messages.po]: This doesn't translate that well into Vietnamese. Maybe consider just removing this if possible?

File: locales/vi/messages.po [hunk 2]
```diff
@@ -78,27 +79,27 @@ msgstr "2:20 PM"
 # the typo here is intentional - please maintain a similar unprofessional
 # tone!
 msgid "example.bad.message2.body"
-msgstr "thứ đo lại là lúc mấy giờ?"
+msgstr "cai đó lúc mấy giờ thế?"
```
Comment (lts20050703) [inline locales/vi/messages.po:82]:
```suggestion
msgstr "cái đó lúc mấy giờ thế?"
```

File: locales/vi/messages.po [hunk 3]
```diff
@@ -78,27 +79,27 @@ msgstr "2:20 PM"
...
-"Nếu bạn cảm thấy hơi khó khăn khi chỉ nói \"Xin chào\" và đặt câu hỏi, **bạn vẫn có thể mở đầu thông điệp của mình với bao nhiêu câu chuyện vui mà bạn thấy phù hợp.**\n"
+"Nếu bạn cảm thấy hơi khó khi chỉ nói \"Xin chào\" và đặt câu hỏi, **bạn vẫn có thể mở đầu tin nhắn của mình với bao nhiêu nhận xét hay mà bạn thấy phù hợp.**\n"
 "\n"
 "Ví dụ:\n"
 "\n"
-"- \"Này anh bạn, có chuyện gì vậy? Còn nữa, bạn có biết khi nào cái đó đến hạn không?\"\n"
-"- \"Chào bạn! Hy vọng bạn khỏe. Tôi đang theo dõi bộ bài mới nhất, khi bạn có một giây :)\"\n"
+"- \"Này anh bạn, có chuyện gì không? À mà, bạn có biết khi nào là hạn của việc đó không?\"\n"
+"- \"Chào bạn! Mong bạn khỏe. Tôi đang theo dõi bộ bài mới nhất, khi bạn rảnh :)\"\n"
 "- \"này, nếu bạn không bận, bạn có thể cập nhật các NFR đó không?\"\n"
```
Comment (lts20050703) [inline locales/vi/messages.po]:
```suggestion
"- \"này, nếu bạn không bận, bạn có thể cập nhật các yêu cầu phi chức năng đó không?\"\n"
```

**Review by lts20050703 — COMMENTED** [review]
Just my 2 cents.
No need to apply all three changes

File: locales/vi/messages.po (hunk 1 above)
Comment (Endy3032) [reply locales/vi/messages.po]: oh yea lemme translate them rq

File: locales/vi/messages.po (hunk 2 above)
Comment (Endy3032) [reply locales/vi/messages.po:82]: the typo there was intentional so this one wont be applied

File: locales/vi/messages.po (hunk 3 above)
Comment (Endy3032) [reply locales/vi/messages.po]: alr great!

**Pull request closed without merging**